src/content/blog/biome-v1-9.md

```diff
@@ -0,0 +1,236 @@
+- [suspicious/noDuplicateAtImportRules](https://biomejs.dev/linter/rules/no-duplicate-at-import-rules/)
+- [suspicious/noDuplicateFontNames](https://biomejs.dev/linter/rules/no-duplicate-font-names/)
+- [suspicious/noDuplicateSelectorsKeyframeBlock](https://biomejs.dev/linter/rules/no-duplicate-selectors-keyframe-block/)
+- [suspicious/noEmptyBlock](https://biomejs.dev/linter/rules/no-empty-block/)
+- [suspicious/noImportantInKeyframe](https://biomejs.dev/linter/rules/no-important-in-keyframe/)
+- [suspicious/noShorthandPropertyOverrides](https://biomejs.dev/linter/rules/no-shorthand-property-overrides/)
+
+If you don't want Biome to format and lint your CSS files,
+you can disable the CSS formatter and linter in the Biome configuration file:
+
+```json
+{
+  "css": {
+    "formatter": {
+      "enabled": false
+    },
+    "linter": {
+      "enabled": false
+    }
+  }
+}
+```
+
+or on the command line:
+
+```shell
+biome format --css-formatter-enabled=false
+biome lint --css-linter-enabled=false
+biome check --css-formatter-enabled=false --css-linter-enabled=false
+```
+
+### Stable GraphQL formatter and linter
+
+Biome now formats and lints GraphQL files by default.
+
+For now, Biome provides two nursery rules:
+
+- [nursery/noDuplicateFields](https://biomejs.dev/linter/rules/no-duplicate-fields/)
+- [nursery/useDeprecatedReason](https://biomejs.dev/linter/rules/use-deprecated-reason/)
+
+If you don't want Biome to format and lint your GraphQL files,
+you can disable the GraphQL formatter and linter in the Biome configuration file:
+
+```json
+{
+  "graphql": {
+    "formatter": {
+      "enabled": false
+    },
+    "linter": {
+      "enabled": false
+    }
+  }
+}
+```
+
+or on the command line:
+
+```shell
+biome format --graphql-formatter-enabled=false
+biome lint --graphql-linter-enabled=false
+biome check --graphql-formatter-enabled=false --css-linter-enabled=false
+```
+
+### Search command
+
+The `biome search` command allows you to search in your project using GritQL syntax.
+
+[GritQL](https://docs.grit.io/language/overview) is a powerful language that lets you do structural searches on your codebase.
+This means that trivia such as whitespace or even the type of strings quotes used will be ignored in your search query.
+It also has many features for querying the structure of your code, making it much more elegant for searching code than regular expressions.
+
+While we believe this command may already be useful to users in some situations (especially when integrated in the IDE extensions!), we also had an ulterior motive for adding this command:
+We intend to utilize GritQL for our plugin efforts, and by allowing our users to try it out in a first iteration, we hope to gain insight in the type of queries you want to do, as well as the bugs we need to focus on.
+
+For now, the search command is explicitly marked as **EXPERIMENTAL**, since many limitations remain.
```

Comment:
```suggestion
For now, the search command is explicitly marked as **EXPERIMENTAL**, since many limitations are yet to be fixed or explored.
```

Reply:
A couple of suggestions:
- we could add an image/html of the diagnostics emitted by the command. Some visual aids can certainly help showing how the search works 
- we could add a special shout out to the contributor that helped us to implement the the search in the CLI

Reply:
Thanks for the suggestions! The blog post is in an early stage. I thought about adding special mention for contributors that helped in different areas. @arendjr don't hesitate to improve this section. I copied your changelog entry as initial content.